canonical/jaas-dashboard
# WD-465 Set up store

## Done

- Set up store and Redux Toolkit.
- Update factories to use the standard factory functions.
- Split up factory files.
- Make UI state a slice.
- Add tests for UI slice (there were none).

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that you can log in a view controllers and models.
